src/ucp/proto/proto_common.h

```diff
@@ -311,4 +311,13 @@ void ucp_proto_reset_fatal_not_implemented(ucp_request_t *req);
 
 void ucp_proto_fatal_invalid_stage(ucp_request_t *req, const char *func_name);
 
+/**
+ * Activate UCP worker interface.
+ *
+ * @param [in] params Protocol initialization parameters.
+ * @param [in] lane   Lane with the resource that the interface corresponds to.
+ */
+void ucp_proto_common_wiface_activate(const ucp_proto_init_params_t *params,
+                                      ucp_lane_index_t lane);
```

Comment:
maybe provide lane_map? could be UCS_BIT(lane) in case of single

Reply:
changed to lane_map